tests/rbd/test_rbd.py

```diff
@@ -75,13 +73,27 @@ def run(ceph_cluster, **kwargs) -> int:
 
     branch = config.get("branch", "pacific")
     nodes = config.get("nodes", [])
+    rhbuild = config.get("rhbuild")
 
     if nodes:
         nodes = get_nodes_by_ids(ceph_cluster, nodes)
     else:
         # By default, tests would be executed on a single client node
         nodes = [ceph_cluster.get_nodes(role="client")[0]]
 
+    if "5." in rhbuild:
+        nodes[0].exec_command("ceph config set mon mon_allow_pool_delete true")
+        nodes[0].exec_command("ceph orch restart mon")
+    else:
+        nodes[0].exec_command(sudo=True, cmd="ceph config set mon mon_allow_pool_delete true")
+        mon_nodes = ceph_cluster.get_nodes(role="mon")
+        for ceph_mon in mon_nodes:
+            ceph_mon.exec_command(
+                sudo=True,
+                cmd="systemctl restart ceph-mon@{}".format(ceph_mon.hostname),
```

Comment:
Opt to use `cmd=f"systemctl restart ceph-mon@{ceph_mon.hostname}"`

Reply:
Done. Made the necessary changes.